saged/sample_split_control.py

```diff
@@ -49,17 +49,22 @@
     parser.add_argument('--weighted_loss',
                         help='Weight classes based on the inverse of their prevalence',
                         action='store_true')
+    parser.add_argument('--sex_label_path',
```

Comment:
So if this argument is included, the label/target variable is sex, and if it's not the target variable is tissue type, correct? Might be worth mentioning that this argument controls whether sex labels are used, and not just where they are located.

Reply:
That's right, and I agree that that's a weird design choice. I'll change it to a boolean flag, then retrieve the path to the labels from the dataset config file